include/openspace/interaction/orbitalnavigator.h

```diff
@@ -0,0 +1,237 @@
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#ifndef __OPENSPACE_CORE___ORBITALNAVIGATOR___H__
+#define __OPENSPACE_CORE___ORBITALNAVIGATOR___H__
+
+#include <openspace/interaction/delayedvariable.h>
+#include <openspace/interaction/inputstate.h>
+#include <openspace/interaction/interpolator.h>
+#include <openspace/interaction/mousestate.h>
+
+#include <openspace/properties/propertyowner.h>
+#include <openspace/properties/scalar/boolproperty.h>
+#include <openspace/properties/scalar/floatproperty.h>
+
+#include <glm/glm.hpp>
+#include <glm/gtx/quaternion.hpp>
+
+namespace openspace {
+
+class SceneGraphNode;
+class Camera;
+class SurfacePositionHandle;
+
+namespace globebrowsing {
+    class RenderableGlobe;
+    class TileIndex;
+    class Geodetic2;
+    class Geodetic3;
+}
+
+namespace interaction {
+
+class OrbitalNavigator : public properties::PropertyOwner 
+{
+public:
+    OrbitalNavigator();
+    ~OrbitalNavigator();
+
+    void updateMouseStatesFromInput(const InputState& inputState, double deltaTime);
+    void updateCameraStateFromMouseStates(Camera& camera, double deltaTime);
+    void setFocusNode(SceneGraphNode* focusNode);
+    void startInterpolateCameraDirection(const Camera& camera);
+
+    bool followingNodeRotation() const;    
+    SceneGraphNode* focusNode() const;
+
+#ifdef OPENSPACE_MODULE_GLOBEBROWSING_ENABLED
```

Comment:
The alternatives are:
1. Remove the possibility to use the functions `goToGeo` and `goToChunk`
2. Move the geodetics to the core and provide an interface to get a Cartesian position given a geodetic coordinate in the `SceneGraphNode`.

I would say alternative 2 is better. I don't think we need to move anything more than the geodetics (ellipsoid can still be in the globebrowsing module). The `goToChunk` might not be as useful but I think `goToGeo` is worth keeping.